Documentation/typo3cms/Developers/Index.rst

```diff
@@ -0,0 +1,196 @@
+.. _resources-developers:
+
+
+.. ------------------------------------------------------------
+.. todos:
+.. Look through the following resources if they are up-to-date
+.. and can be used or update / migrate / delete them
+..
+.. - Wiki: https://wiki.typo3.org/Extension_Developers_Guide
+.. - Wiki: https://wiki.typo3.org/Extension_Development
+.. -------------------------------------------------------------
+
+========================
+Resources for Developers
+========================
+
+This page contains some resources for TYPO3 developers including information
+about **extension development**, **site package extension development** and
+**core development**.
+
+It contains links to official TYPO3 docs. However, if the official docs are
+outdated and there are newer resources available elsewhere (e.g. on blogs),
+a link to these is supplied.
+
+You can assume that whatever starts with
+`http://docs.typo3.org/typo3cms` and is linked from this page is official
+TYPO3 documentation.
+
+.. _resources-developers-extension:
+
+Extension Development
+=====================
+
+.. _resources-developers-extension-getting-started:
+
+Getting started
+---------------
+
+For getting started with TYPO3 development, please ask for up-to-date
+resources in the Slack channel **#typo3-cms** or **#typo3-documentation**
+(register for Slack here: https://my.typo3.org/)
+
+It is difficult to give general information here because it depends on the
+TYPO3 version you will be using and what you plan to do.
+
+.. _resources-developers-extension-getting-started-extbase-fluid:
+
+Extbase / Fluid
+~~~~~~~~~~~~~~~
+
+.. todo: question: can these resources still be recommended for TYPO3 8?
+.. should additional information be added here?
+
+For a walkthrough guide on Extension Development with Extbase / Fluid
+you can look at:
+
+* :ref:`Developing TYPO3 Extensions with Extbase and Fluid <t3extbasebook:start>`
+  (Official TYPO3 docs, a translation of the German book by Kurfürst and  Rau, updated
+  by the community)
+* The (hardcover) book: Michael Schams and Patrick Lobacher: The TYPO3 Extbase Book
+  (http://extbase-book.org) The latest version is for TYPO3 7.6.
+
+.. todo: insert general informational text about current status of Extbase
+
+
+Fluid
+~~~~~
+
+Note that Fluid is now maintained outside of the TYPO3 core. The core itself
+contains an extension named EXT:fluid with some TYPO3-specific functionality and
+there is a composer dependency on typo3fluid/fluid, which is the main Fluid
```

Comment:
I would suggest to link `typo3fluid/fluid` as text to https://packagist.org/packages/typo3fluid/fluid
As we are writing about a composer dependency, we can link this and provide a better UX to the user.